client/components/ReviewsComponent.jsx

```diff
@@ -15,9 +15,14 @@ class App extends React.Component {
   }
 
   componentDidMount() {
+
     this._isMounted = true;
 
-    fetch('http://localhost:3007/api/userReviews/1')
+    let urlParts = window.location.href.split('/');
+    let Id;
+    urlParts[urlParts.length - 1] === '' ? Id = 1 : Id = urlParts[urlParts.length - 1];
+
+    fetch(`http://localhost:3007/api/userReviews/${Id}`)
```

Comment:
Nice job here! Just a thought for testing - unless you're already using `jsdom`, you may need a check if `window` exists. If you're already using jsdom, no need to make changes here.

Reply:
Interesting! I'm still behind on getting more tests, but I'll make that change just to avoid any issues down the road.